package/MDAnalysis/analysis/hydrogenbonds/hbond_analysis.py

```diff
@@ -206,6 +209,14 @@ def __init__(self, universe, donors_sel=None, hydrogens_sel=None, acceptors_sel=
         acceptors_sel : str
             Selection string for the hydrogen bond acceptor atoms. Leave as `None` to guess which atoms to use in the
             analysis using :attr:`guess_acceptors`
+        between : List (optional)
```

Comment:
Add an example to the docs above that shows how to use “between”, essentially, take the text here and add code.